docs/source/workflow_inputs.rst

```diff
@@ -1154,6 +1156,17 @@ If there is a cooling temperature setup, additional information is entered in ``
   ``extension/SetupStartHourCooling``    integer             0 - 23       No        9 (9am)    Daily setup start hour
   =====================================  ========  ========  ===========  ========  =========  =========================================
 
+If a heating and/or cooling season is defined, additional information is entered in ``HVACControl/HeatingSeason`` and/or ``HVACControl/CoolingSeason``.
+
+  ======================================  ========  =====  =================  ========  =============================  ===========
+  Element                                 Type      Units  Constraints        Required  Default                        Description
+  ======================================  ========  =====  =================  ========  =============================  ===========
+  ``BeginMonth`` and ``BeginDayOfMonth``  integer          1 - 12 and 1 - 31  No                                       Start date
+  ``EndMonth`` and ``EndDayOfMonth``      integer          1 - 12 and 1 - 31  No                                       End date
```

Comment:
Split these up into 4 rows, one for each element. And they should all be marked as required.